### PR TITLE
docs: add faq question for the dashboard timeout issue

### PR DESCRIPTION
**Pull request by nicufk**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

**Review by vLia — APPROVED** [review]

File: docs/docs/12-FAQ.md [hunk 1]
```diff
@@ -24,6 +24,24 @@ pod/testkube-mongodb-74587998bb-8pzl2                       1/1     Running   0
 pod/testkube-operator-controller-manager-77ffbb8fdc-rxhvx   2/2     Running   0          5d23h
 ```
 
+### Dashboard opening using cli is not working
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:27]:
```suggestion
### Opening the Dashboard using CLI is not working.
```

File: docs/docs/12-FAQ.md [hunk 2]
```diff
@@ -24,6 +24,24 @@ pod/testkube-mongodb-74587998bb-8pzl2                       1/1     Running   0
...
+
+If there is a timeout error reported
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:29]:
```suggestion
If there is a timeout error reported:
```

File: docs/docs/12-FAQ.md [hunk 3]
```diff
@@ -24,6 +24,24 @@ pod/testkube-mongodb-74587998bb-8pzl2                       1/1     Running   0
...
+
+```sh
+kubectl-testkube dashboard
+checking readiness of services (error: timed-out waiting for dashboard and api)
+```
+
+Make sure that there is not another dashboard process running in the background like in the example bellow
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:36]:
```suggestion
Make sure that there is not another dashboard process running in the background like in the example below:
```

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

**Pull request merged**